src/platform/user/profile/vap-svc/components/ContactInformationFieldInfo/ContactInformationUpdateSuccessAlert.jsx

```diff
@@ -37,7 +37,12 @@ const ContactInformationUpdateSuccessAlert = ({ fieldName }) => {
         return (
           <>
             Update saved. Now you can{' '}
-            <Link to={PROFILE_PATHS.NOTIFICATION_SETTINGS}>
+            <Link
+              to={{
+                pathname: PROFILE_PATHS.NOTIFICATION_SETTINGS,
+                state: { scrollToTop: true },
```

Comment:
TIL you can pass state to a react router Link component that will be available in the destination route with the `useLocation()` hook as `location.state`